class DataError(Exception):
    pass

class RecordValidationError(DataError):
    pass
